lib/order.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'menu'
+
+class Order
+
+  attr_reader :order, :menu, :total
+
+  def initialize
+    @order = []
+    @total = 0
+    @menu = Menu.new
+  end
+
+  def add_to_basket(food, quantity = 1)
+    quantity.times { @order << food }
+  end
+
+  def checkout
+    @order.map do |food|
+      @total += @menu.dish[food] 
+    end
```

Comment:
In the rubric &  user story 3 they suggested adding a set so the use could check it is the right amount